packages/react-native/project.json

```
@@ -13,6 +13,8 @@
         "assets": [
           "packages/react-native/*.md",
           "packages/react-native/index.d.ts",
+          "packages/react-native/React.podspec",
+          "packages/react-native/React-Core.podspec",
```

As discussed, if we move the podspecs into `platforms/ios`, this should become (and I'm adding in `Podfile` here out of caution, though I'm not sure whether NativeScript's nx configuration implicitly includes it for us already):

```suggestion
          "packages/react-native/platforms/ios/Podfile",
          "packages/react-native/platforms/ios/React.podspec",
          "packages/react-native/platforms/ios/React-Core.podspec",
```